### ISSUE TITLE
Inconsistencies between timestamp usage in server python scripts

### ISSUE THREAD

**Issue by ndokos** [distributed-system-analysis/pbench#1252]
The server scripts log the beginning and end of their run with a timestamp and also pass a timestamp to `report_status()` to send their status to ES. Currently, we have different behaviors wrt to these timestamps:

- the bash scripts:
  - log beginning using `TS`
  - log end using `TS` followed by summary
  - `report_status` using `TS`

- pbench-index.py:
  - log beginning using `idxctx.TS = config.TS = TS`
  - log end using `idxctx.TS = config.TS = TS`
  - `report_status` using `idxctx.time()`

- pbench-backup-tarballs.py:
  - log beginning using `config.timestamp()`
  - log end using using `config.timestamp()`
  - `report_status` using `config.timestamp()`

- pbench-verify-backup-tarballs.py:
  - log beginning using `config.TS`
  - log end using `config.TS`
  - `report_status` using `config.timestamp()`

Basically, TS = config.TS = idxctx.TS = fixed timestamp set at the beginning of the run. It is used as an identifier for a single run in the logs, so if multiple items are logged they are all tagged with this.

These should all do the same things:
 
- log the beginning of the run using `TS = config.TS = idxctx.TS` 
- log the end of the run using  `TS = config.TS = idxctx.TS` followed by summary (if applicable)
- use `config.timestamp()` or the `timestamp` function from pbench-base.sh when calling `report_status()`
